src/saveload/de.rs

```diff
@@ -0,0 +1,146 @@
+
+use std::fmt::{self, Display, Formatter};
```

Comment:
I'd prefer either just `use std::fmt` or no `self` in the list for consistency.

Reply:
Why don't you like `self` here?

Reply:
I just find it inconsistent that some of the types in the same namespace are referred directly while others - via `fmt::Type`. I believe it reads better through the code if all the imports from the same namespace are consistent. In case of `fmt`, I find `fmt::Display` and `fmt::Formatter` quite nice, but that's a matter of taste.